.github/workflows/repo_linter.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# by default github actions doesn't fetch the whole repo so we are getting the latest commit on main
+git fetch origin main:main --depth=1
```

Comment:
This can be set in the workflow file.

Reply:
I didn't know that. Thanks for sharing that's very handy.